<ui version="4.0" >
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>1096</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Delta</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../media/mainwindow.qrc" >
    <normaloff>:/MainWindow/logoDelta.png</normaloff>:/MainWindow/logoDelta.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget" >
   <layout class="QGridLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <item row="0" column="0" >
     <widget class="Canvas" native="1" name="mainCanvas" />
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen" />
    <addaction name="actionOpenDefaultScene" />
    <addaction name="separator" />
    <addaction name="actionLoadCamera" />
    <addaction name="actionSaveCamera" />
    <addaction name="separator" />
    <addaction name="actionExit" />
   </widget>
   <widget class="QMenu" name="menuRender_Mode" >
    <property name="title" >
     <string>&amp;Rendering</string>
    </property>
    <addaction name="actionRedrawAlways" />
    <addaction name="actionReloadShaders" />
    <addaction name="actionCanvasSize" />
   </widget>
   <addaction name="menu_File" />
   <addaction name="menuRender_Mode" />
  </widget>
  <widget class="QToolBar" name="mainToolBar" >
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen" />
   <addaction name="actionLoadCamera" />
   <addaction name="actionSaveCamera" />
   <addaction name="separator" />
   <addaction name="actionRedrawAlways" />
   <addaction name="actionReloadShaders" />
   <addaction name="actionCanvasSize" />
   <addaction name="separator" />
  </widget>
  <widget class="QStatusBar" name="statusBar" />
  <action name="actionExit" >
   <property name="text" >
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionRedrawAlways" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../media/mainwindow.qrc" >
     <normaloff>:/MainWindow/redrawAlways.png</normaloff>:/MainWindow/redrawAlways.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Continuous Redraw</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionReloadShaders" >
   <property name="text" >
    <string>&amp;Reload Shaders</string>
   </property>
   <property name="shortcut" >
    <string>F5</string>
   </property>
  </action>
  <action name="actionSingleCpu" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>&amp;Single CPU</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionMultiCpu" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>&amp;Multi CPU</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionGlsl" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>&amp;GLSL</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="actionOpen" >
   <property name="icon" >
    <iconset resource="../media/mainwindow.qrc" >
     <normaloff>:/MainWindow/fileOpen.png</normaloff>:/MainWindow/fileOpen.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Open model...</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionCuda" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>&amp;CUDA</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+4</string>
   </property>
  </action>
  <action name="actionLoadCamera" >
   <property name="text" >
    <string>Load camera</string>
   </property>
  </action>
  <action name="actionSaveCamera" >
   <property name="text" >
    <string>Save camera</string>
   </property>
  </action>
  <action name="actionOpenDefaultScene" >
   <property name="text" >
    <string>Open default scene</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionCanvasSize" >
   <property name="text" >
    <string>Canvas size...</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+C</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>Canvas</class>
   <extends>QWidget</extends>
   <header>Canvas.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../media/mainwindow.qrc" />
  <include location="../media/mainwindow.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
